.prettierrc

```
@@ -0,0 +1,16 @@
+{
+
+    "printWidth": 90,
+    "tabWidth": 4,
+    "useTabs": false,
+    "semi": false,
```

```suggestion
    "semi": true,
```
This is to avoid ASI failures: https://prettier.io/docs/en/rationale.html#semicolons.